packages/permission-controller/src/PermissionController.test.ts

```diff
@@ -1,5 +1,5 @@
 import assert from 'assert';
-import { JsonRpcEngine, PendingJsonRpcResponse } from 'json-rpc-engine';
+import { JsonRpcEngine } from '@metamask/json-rpc-engine';
```

Comment:
Also this will make the PermissionController minimum Node 16. Are we ready for that in this repo?

Reply:
Good catch @FrederikBolding. 

I don't know if we can support bumping the minimum Node version for select packages in this monorepo because the CI matrix is maintained across the whole monorepo, so running tests for packages that support >= 16 would fail on 14.x. We'd either need to get the GitHub workflows to accommodate this or bump the Node version across all of the packages.

(That said, the `@metamask/permission-controller` test job _isn't_ failing on 14.x with this change, so that's curious...)

Either way, it might be good to do this in another PR for visibility.

Reply:
Bumping `@metamask/json-rpc-engine` requires bumping `@metamask/utils` and `@metamask/rpc-errors`, due to type conflicts with `@metamask/types` otherwise. I could cherry-pick the first commit into a separate PR, if that makes it more clear?

Reply:
As of today bumping this library to 16 should not be a problem as mobile is now running Node 16.

Reply:
@mrtenz Cherry-picking the first commit in a separate PR makes sense. I think it's also be good to make a new PR that bumps the minimum version of Node to 16 across the board so we can codify that in the next release.